hadoop-ozone/dist/src/main/k8s/examples/regenerate-all.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -e
+
+if [ ! -x "$(command -v flekszible)" ]; then
+    echo "flekszible tool is required for regenerating Kubernetes files"
+    exit 1
+fi
+
+SCRIPT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )
+cd "$SCRIPT_DIR"
+for dir in $(find . -maxdepth 1 -mindepth 1 -type d -printf '%f\n'); do
```

Comment:
On macosx, the find command is different with Linux, 

```bash
✗ find
usage: find [-H | -L | -P] [-EXdsx] [-f path] path ... [expression]
       find [-H | -L | -P] [-EXdsx] -f path [path ...] [expression]
```

Reply:
It cannot work without flekszible.yaml fiie


```bash
➜  getting-started git:(fixcsi3) ✗ flekszible generate  
INFO[2020-06-25T09:17:36+08:00] Input dir: /Users/mbl/projects/github/hadoop-ozone/hadoop-ozone/dist/src/main/k8s/examples/getting-started, output dir: /Users/mbl/projects/github/hadoop-ozone/hadoop-ozone/dist/src/main/k8s/examples/getting-started 
INFO[2020-06-25T09:17:36+08:00] Reading resources from /Users/mbl/projects/github/hadoop-ozone/hadoop-ozone/dist/src/main/k8s/examples/getting-started/resources 
2020/06/25 09:17:36 Can't parse flekszible.yaml/Flekszible descriptor from  /Users/mbl/projects/github/hadoop-ozone/hadoop-ozone/dist/src/main/k8s/examples/getting-started/flekszible/ozone: open /Users/mbl/projects/github/hadoop-ozone/hadoop-ozone/dist/src/main/k8s/examples/getting-started/flekszible/ozone/flekszible.yaml: not a directory

```

Reply:
Thanks the review @maobaolong

> On macosx, the find command is minor different with Linux, so is there another tool can replace find?

Yes, I started with a simple `ls`, but shellcheck complained that it couldn't work with space in the dir names (which not supposed to be used). Fixed back.

> It cannot work without flekszible.yaml fiie

It's a bug in a recent addition in flekszible. Can you please use [1.8.0 version](https://github.com/elek/flekszible/releases/tag/v1.8.0) which contains the fix. (OSX test is added to Flekszible to avoid similar issues in the future)

Reply:
@elek Thank you for your changes, it works on my macosx.

Reply:
Alternative (for next time):

```
find * -maxdepth 0 -mindepth 0 -type d
```